README.md

```diff
@@ -86,22 +86,27 @@ For a start we made the following choices:
     experiment_name: my_custom_experiment
     n_repeats: 5
     scenario_params_list:
-     - partners_count: 3
-       amounts_per_partner: [0.4, 0.3, 0.3] 
-       samples_split_option: 'random'
-       aggregation_weighting: 'data_volume'
-       single_partner_test_mode: 'global'
-       epoch_count: 38
-       minibatch_count: 20
-     - partners_count: 4
-       amounts_per_partner: [0.3, 0.3, 0.1, 0.3] 
-       samples_split_option: 'stratified'
-       aggregation_weighting: 'data_volume'
-       single_partner_test_mode: 'global'
-       epoch_count: 38
-       minibatch_count: 20
+         partners_count: 
```

Comment:
We need to update the README since this config file does not work anymore (with different `partners_count `in the same scenario block)